pkg/tree/tree.go

```diff
@@ -0,0 +1,120 @@
+package tree
+
+import (
+	"github.com/ghodss/yaml"
+)
+
+// Tree is a structured representation of a string keyed tree document such as
+// yaml or json.
+type Tree map[string]interface{}
```

Comment:
You down with ADT? Yeah you know me!
You down with ADT? Yeah you know me!
You down with ADT? Yeah you know me!
Who's down with ADT?  **NOT GOLANG**